src/main/java/javaintern/constant/ReservationState.java

```diff
@@ -0,0 +1,5 @@
+package javaintern.constant;
+
+public enum ReservationState {
+	Initial, Confirmed, FullyPaid, Cancelled
```

Comment:
It's better to use UPPERCASE in enum naming

Reply:
vasylnikorych told me to use CamelCase
vasylnikorych [Nov 4, 2022]
change style according:
State of record (Initial, Confirmed, FullyPaid, Cancelled)